ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+**Issue Template**
+*Title* : *One line description*
```

Comment:
nit: space on either side of ':' is pretty strange. Can we just align ':' with the thing to the left of it.

Reply:
(same throughout entire change)